source/openapi-admin-v3.yaml

```
@@ -5525,7 +5547,71 @@ components:
         error_code:
           type: string
           description: The error type.
-    AtlasCluster:
+    AtlasConfigResponse:
+      type: object
+      properties:
+        config:
+          type: object
+          properties:
+            clusterId:
+              type: number
+              example: 05e821b3975d271289f372e3a
+              description: The service ID.
+            clusterName:
+              type: string
+              example: Cluster0
+              description: |-
+                The service name used to refer to the data source within
+                this Atlas app. The name may be at most 64 characters
+                long and must only contain ASCII letters, numbers,
+                underscores, and hyphens.
+            clusterType:
+              type: string
+              example: atlas
+              description: The type of service.
```

```suggestion
            clusterType:
              type: string
              enum:
                - atlas
                - serverless
              example: atlas
              description: The type of service.
```

There's only two options here so we can use an enum